### ISSUE TITLE
Question about remove

### ISSUE THREAD

**Issue by srobb1** [ontodev/robot#573]
Hello All,
I have two relationship terms that are causing me issues from imports when I use reasoning. If I delete these two terms using protege (highlight term and click the delete button) then "start reasoner", all is good.

I am trying to do this atomically with robot. Below is my command. When I run this the terms are indeed removed from my ontology but when I open the simr-clean and try to reason I get crazy errors.
```
robot remove  --input simr.owl -t CLO:0000015 -t OBI:0000293 --select "self descendants" --signature true --output simr-clean.owl
```
I have also tried
```
robot remove  --input simr.owl -t CLO:0000015 -t OBI:0000293   --output simr-clean.owl
```
as well as variety of other variations.

What robot command can I use to replicate the protege delete?

Thank you,
Sofia

**Comment by beckyjackson** [comment]
Would you mind sharing the file that you're using so I can recreate this issue? Thanks!

**Comment by srobb1** [comment]
Thank you for helping with this. Here is my owl file:

[simr.owl](https://github.com/srobb1/simro/blob/master/simro.owl)

Thanks!
Sofia

**Comment by beckyjackson** [comment]
Hi again - it looks like there's something weird going on with `--preserve-structure true` and the object properties. I'm going to look into this, but for now, you should be able to get a clean output by adding `--preserve-structure false` (or `-p false`). Since you're just removing those two object properties, it shouldn't make a difference for the hierarchy.

**Comment by srobb1** [comment]
Hi @beckyjackson Thanks!! It seems like it is working, but any idea why I get 
owl:Nothing is_a owl:Nothing after I reason? By the way, I am using elk-0.5.0.snapshot.

![Screen Shot 2019-10-03 at 9 44 17 AM](https://user-images.githubusercontent.com/1072991/66142480-b1263c80-e5c2-11e9-8c18-7dde00662f77.png)

**Comment by balhoff** [comment]
@srobb1 if you are using the latest ROBOT, try adding `--exclude-tautologies structural` to get rid of that.

**Comment by srobb1** [comment]
Hi @balhoff. I have `ROBOT version 1.4.3` when I run `robot remove -h` I do not see that argument in the list.

**Comment by balhoff** [comment]
Sorry—this is an option for `reason`.

**Comment by srobb1** [comment]
@balhoff  I tried using my owl with the ObjProps removed by `robot remove` as input to `robot reason` with `--exclude-tautologies structural` and I still get the double nothing. It isn't pretty, but does it hurt anything to have this weirdness?

**Comment by jamesaoverton** [comment]
The extra Nothing sounds like this: https://github.com/ontodev/robot/issues/562#issuecomment-537060674

**Comment by beckyjackson** [comment]
This issue was resolved by #575 and the `owl:Nothing` issue was resolved in #579. You can remove `owl:Nothing` with:
```
robot remove --input input.owl \
  --axioms structural-tautologies \
  --output output.owl
```
